qa/common/nightly_email_helper.py

```diff
@@ -39,7 +39,7 @@
 
 def send(subject: str,
          content: str,
-         attachments=[],
```

Comment:
Why is this change needed?

Reply:
It's because of the attachments.append. Default arguments are instantiated once, not once per call. By using .append, the default argument is changed. So the behavior would vary for every call (i.e. once a new attachment is appended when using the default value, that attachment would already be appended for the next call with the default attachments value).

Reply:
TIL 😄